foundation_cms/static/scss/components/donate_banner/donate_banner.scss

```diff
@@ -17,6 +17,11 @@
   position: relative;
   padding-top: calc(map.get($grid-container-padding, small) / 2);
   padding-bottom: calc(map.get($grid-container-padding, small) / 2);
+  display: none;
```

Comment:
This `display: none;` hides all banner versions by default. Since the `legacy` version needs special treatment, we should target it specifically with this rule. Alternatively, we could keep all banners visible by default (no SCSS changes) and use JS to call `banner.remove()` when the legacy banner shouldn't be shown. This approach avoids additional operations like:
- Hiding then showing the pushdown version
- Removing an already-hidden legacy banner